src/file.rs

```diff
@@ -26,3 +31,9 @@ fn write(data: &str, path: &str) -> Result<usize> {
 
     Ok(file.write(data.as_bytes())?)
 }
+
+fn append(data: &str, path: &str) -> Result<usize> {
+    let file = OpenOptions::new().append(true).create(true).open(path)?;
```

Comment:
Needs to be mutable for the `write()` call to work:
```suggestion
    let mut file = OpenOptions::new().append(true).create(true).open(path)?;
```
I'll tweak Travis to catch this.